res/css/structures/_ToastContainer.scss

```diff
@@ -91,10 +91,7 @@ limitations under the License.
         }
 
         .mx_Toast_description {
-            max-width: 400px;
-            overflow: hidden;
-            white-space: nowrap;
```

Comment:
could maybe do with a word wrap of some kind to prevent the third screenshot from happening.

Reply:
Have put the overflows back - screenshot updated